source/internal-communication.rst

```diff
@@ -557,8 +557,48 @@ certificate-related services for devices.
 Kafka-WS
```

Comment:
I think you can change  "Kafka Websocket" to "Kafka WS (Kafka Websocket)" or just "Kafka WS"

Reply:
I think it's nice. I vote for `Kafka WS`!